specification/sections/01-flag-evaluation.md

```diff
@@ -22,7 +22,7 @@ It's important that multiple instances of the `API` not be active, so that state
 
 #### Requirement 1.1.2
 
-> The `API` **MUST** provide a function to set the global `provider` singleton, which accepts an API-conformant `provider` implementation.
+> The `API` **MUST** provide a function to set the default `provider`, which accepts an API-conformant `provider` implementation.
```

Comment:
I think it might be nice to add some non-normative text below this about how the `default` provider is different from `client-name` bound providers.